core/domain/html_validation_service.py

```diff
@@ -850,20 +853,108 @@ def add_dimensions_to_image_tags(exp_id, html_string):
     return python_utils.UNICODE(soup).replace('<br/>', '<br>')
 
 
-def get_filename_with_dimensions(old_filename, exp_id):
+def add_dimensions_to_image_tags_inside_tabs_and_collapsible_blocks(
```

Comment:
Hmmm, this wouldn't necessarily belong in a "validation_service" right? It seems to be modifying and not verifying.